Source/Core/InputCommon/ControllerEmu.h

```diff
@@ -50,27 +51,27 @@ class ControllerEmu
     class Control
     {
     protected:
-      Control(ControllerInterface::ControlReference* const _ref, const std::string& _name)
+      Control(ControlReference* const _ref, const std::string& _name)
```

Comment:
Names that start with an underscore are reserved by the standard, can you rename those while you're at it. Easiest would probably be to fix the member variables too (`m_` prefix) so you can use their names here.